File: AtvdExtra2/controleDeAudio.java
//ControleDeAudio.java


public class ControleDeAudio {
    private int volume = 5; 

    public int aumentarVolume() {
        return this.volume < 10 ? this.volume++ : this.volume;
    }

    public int diminuirVolume() {
        return this.volume > 0 ? this.volume-- : this.volume;
    }

    public int lerVolume() {
        return volume;
    }

    public static void main(String[] args) {
        ControleDeAudio controle = new ControleDeAudio();

        for (int i = 1; i < 10; i++) {
            controle.aumentarVolume();
        }

        System.out.println(controle.lerVolume()); 
    }
}

File: AtvdExtra2/controleDeAudio.py
class ControleDeAudio: 
    def __init__(self, volume=5): 
        self.volume = volume
        
    def aumentarVolume(self): 
        if self.volume < 10:
            self.volume += 1
            
    def diminuirVolume(self): 
        if self.volume > 0:
            self.volume -= 1
            
    def lerVolume(self):
        return self.volume
        
controle = ControleDeAudio() 

for i in range(10):
    controle.diminuirVolume()

print(controle.lerVolume())
